snap/snapcraft.yaml

```diff
@@ -0,0 +1,204 @@
+name: colmap
+version: '3.0'
+summary: COLMAP - Structure-from-Motion and Multi-View Stereo (colmap.github.io)
+description: |
+  COLMAP is a general-purpose Structure-from-Motion (SfM) and
+  Multi-View Stereo (MVS) pipeline with a graphical and command-line
+  interface. It offers a wide range of features for reconstruction of
+  ordered and unordered image collections.
+
+#grade: devel
+#confinement: devmode
+
+grade: stable
+confinement: strict
+
+#icon: project/src/images/icon.png
+
+apps:
```

Comment:
It seems like not all binaries are included here. One missing file would be the ``automatic_reconstructor``.